openmicroscopy/ome-help
# Update Help site to Version 5.0.8

To test:

Check version numbers are correct (5.0.8) on:
- Home Page - also swapped order of 4 and 5 in text
- Menu entries - Getting Started 5, Importing 5
- Document titles - Getting Started 5, Importing 5
- Training Materials
- Resources
- check screenshots displaying in Virtual Microscope

Note: PDFs from downloads links will not be updated yet - these are on staging.downloads.
